diff --git a/colors/seoul256.vim b/colors/seoul256.vim
index eb238d5..8adb2aa 100644
--- a/colors/seoul256.vim
+++ b/colors/seoul256.vim
@@ -190,8 +190,9 @@ call s:hi('ColorColumn', ['', ''], [s:dark_bg - 1, s:light_bg - 2])
 " hi CursorIM ctermfg=
 
 " set cursorline cursorcolumn
-call s:hi('CursorLine', ['', ''], [s:dark_bg - 1, s:light_bg - 1])
-call s:hi('CursorLineNr', [131, 131], [s:dark_bg - 1, s:light_bg - 1])
+call s:hi('Cursor', ['', ''], [s:dark_bg - 1, s:light_bg - 1])
+call s:hi('CursorLine', ['', ''], [s:dark_bg + 6, s:light_bg - 1])
+call s:hi('CursorLineNr', [131, 131], [s:dark_bg + 6, s:light_bg - 1])
 call s:hi('CursorColumn', ['', ''], [s:dark_bg - 1, s:light_bg - 1])
 
 call s:hi('Directory', [187, 95], ['', ''])
